soc/arc/snps_arc_hs4xd/Kconfig.defconfig

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2019 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_ARC_HS4XD
+
+config SOC
+	default "snps_arc_hs4xd"
```

Comment:
snps_arc_hs4xd -> snps_arc_hsdk4xd

Reply:
SoC based on HS4x core. "DK" as I understand means "Development Kit". Is it correct name SoC with "DK" abbreviation?